modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.segments.internal.upgrade.v2_6_0;
+
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.segments.constants.SegmentsExperienceConstants;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Eudaldo Alonso
+ */
+public class SegmentsExperienceUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_updateSegmentsExperiencePriorities();
+	}
+
+	private void _updateSegmentsExperience(
+		long segmentsExperienceId, int priority) {
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				"update SegmentsExperience set priority = ? where " +
+					"segmentsExperienceId = ?")) {
+
+			preparedStatement.setInt(1, priority + 1);
+			preparedStatement.setLong(2, segmentsExperienceId);
+
+			preparedStatement.executeUpdate();
+		}
+		catch (Exception exception) {
+			if (_log.isDebugEnabled()) {
```

Comment:
Do you want this error is shown only in debug mode? This is a new rule for upgrade processes?

Reply:
Hey @cgoncas,

I just copied the Upgrade process from com.liferay.segments.internal.upgrade.v2_0_0.SegmentsExperienceUpgradeProcess

If you want, we can change both logs and replace for _log.error

Reply:
Please let me know and I send you a follow up pull